gradle/fips.gradle

```diff
@@ -9,8 +9,9 @@ if (BuildParams.inFipsJvm) {
   allprojects {
     File fipsResourcesDir = new File(project.buildDir, 'fips-resources')
     boolean java8 = BuildParams.runtimeJavaVersion == JavaVersion.VERSION_1_8
-    File fipsSecurity = new File(fipsResourcesDir, "fips_java${java8 ? '8' : ''}.security")
-    File fipsPolicy = new File(fipsResourcesDir, "fips_java${java8 ? '8' : ''}.policy")
+    boolean zulu8 = java8 && BuildParams.jvmDetails.contains("Zulu")
```

Comment:
Is the zulu policy the exact same as the java 11, or a mix of our existing java 11 and java 8 policies?

Reply:
The zulu policy and properties are the same as the java 11 with regards to the actual settings and policy rules , but with the format that is applicable for JDK8. Maybe fips_java8 and fips_java_zulu8 are not applicable names here, I will change them to reflect the crypto provider that is in use in each case so that this is clearer